src/Compilers/CSharp/Test/Syntax/Syntax/LambdaUtilitiesTests.cs

```diff
@@ -43,8 +43,10 @@ void M()
             bool expected = enclosingMethod.MethodKind == MethodKind.LambdaMethod && enclosingSyntax.Span.Contains(span.Value);
 
             var node = tree.GetRoot().FindNode(span.Value);
-            Assert.Equal(expected, LambdaUtilities.IsLambdaBody(node));
-            Assert.Equal(isLambdaBody, expected);
+            Assert.False(isLambdaBody && isReducedLambdaBody);
+            Assert.Equal(expected, LambdaUtilities.IsLambdaBody(node, allowReducedLambdas: true));
+            Assert.Equal(isLambdaBody || isReducedLambdaBody, expected);
```

Comment:
Actually it looks like ```expected``` is actual. Consider renaming.

---
In reply to: [92516240](https://github.com/dotnet/roslyn/pull/15868#discussion_r92516240) [](ancestors = 92516240)